src/components/pages/Navbar/Navbar.js

```diff
@@ -82,22 +67,34 @@ const Navbar = ({ isAuthenticated, userProfile }) => {
             </Link>
             {Object.keys(user).length && (
               <div className="userInfo-and-profilePic">
-                <Dropdown overlay={memosMenu} placement="bottomLeft" arrow>
-                  <Link key="memosLinkNav" to="/notes">
-                    Memos
-                  </Link>
-                </Dropdown>
-                <Dropdown overlay={accountMenu} placement="bottomLeft" arrow>
+                <Link
+                  key="memosLinkNav"
+                  to="/notes"
+                  style={{ color: '#FFF' }}
+                  className="memos"
+                >
+                  <FormOutlined className="memo-icon" />
+                  Memos
+                </Link>
+                <Dropdown overlay={accountMenu} placement="bottom" arrow>
                   <div className="userInfo-and-profilePic">
                     <div className="profilePic">
                       <Avatar
                         size={50}
                         icon={<UserOutlined />}
                         src={profilePic}
+                        alt="Account settings"
```

Comment:
I like how you added the alt tags for screen readers. Thanks for demonstrating this in the video. It was really helpful seeing it in action.